contracts/Uniswap/UniswapAnchoredView.sol

```diff
@@ -57,10 +58,10 @@ contract UniswapAnchoredView is UniswapConfig {
     event ReporterInvalidated(address reporter);
 
     /// @notice The event emitted when the uniswap window changes
-    event UniswapWindowUpdate(address indexed uniswapMarket, uint oldTimestamp, uint newTimestamp, uint oldPrice, uint newPrice);
+    event UniswapWindowUpdate(bytes32 indexed symbolHash, uint oldTimestamp, uint newTimestamp, uint oldPrice, uint newPrice);
 
     /// @notice The event emitted when anchor price is updated
-    event AnchorPriceUpdate(address indexed uniswapMarket, uint anchorPrice, uint nowCumulativePrice, uint oldCumulativePrice, uint oldTimestamp);
+    event AnchorPriceUpdate(bytes32 indexed symbolHash, uint anchorPrice, uint nowCumulativePrice, uint oldCumulativePrice, uint oldTimestamp);
```

Comment:
actually, does this event even need cumulative prices? we can always figure out the cumulative prices from `WindowUpdate` events.

Reply:
WindowUpdate is not always emitted

Reply:
but its emitted on every change. why do we need to know these values if they arent changing?

Reply:
Yeah, maybe not